libretroshare/src/jsonapi/jsonapi.cpp

```diff
@@ -395,7 +395,7 @@ JsonApiServer::JsonApiServer(): configMutex("JsonApiServer config"),
 				} );
 			};
 
-			bool retval = rsEvents->registerEventsHandler(
+			auto retval = rsEvents->registerEventsHandler(
```

Comment:
This is typically a place where auto is not good: it's just as long as the type you're hiding, but it keeps you from seeing what the type should be. That's what I'm talking about when I say that sometimes you use C++11 for the sake of using C++11. Here it's really a weird idea.

Reply:
I changed it because it returns std::error_condition now, but I can make it explicit if you like